### PR TITLE
Adding systems-information workload and chart

### PR DESCRIPTION
**Pull request by mattmattox**

### CODE REVIEW

**Comment by mattmattox** [comment]
Here is an example helm install.
```
helm install --replace systems-information ./systems-information-0.0.1-rc1.tgz \
--set rancher_name="rancher-a1" \
--set smtp_host="mail.support.tools" \
--set smtp_port="587" \
--set smtp_user="rancher@support.tools" \
--set smtp_pass="Cattle2020!!" \
--set to_address="matthew.mattox@rancher.com" \
--set from_address="rancher@support.tools" \
--set send_to_support="true"
```

**Comment by dnoland1** [comment]
Sorry, nevermind, see field further above.

**Comment by dnoland1** [comment]
Job is failing to run. Logs:
```
Running Summary Report...

Rancher Systems Summary Report

==============================

Run on Sun Jun 21 23:13:13 UTC 2020

Defaulting container name to rancher.

Use 'kubectl describe pod/rancher-bf4888754-4fkzq -n cattle-system' to see all of the containers in this pod.

Rancher version: v2.4.4

Defaulting container name to rancher.

Use 'kubectl describe pod/rancher-bf4888754-4fkzq -n cattle-system' to see all of the containers in this pod.

Rancher id: c73499c7-95cb-4166-8a66-e641d4828929

Defaulting container name to rancher.

Use 'kubectl describe pod/rancher-bf4888754-4fkzq -n cattle-system' to see all of the containers in this pod.

Cluster Id   Name             K8s Version    Provider   Created                Nodes

c-g8l7q      d-testing        <none>                    2020-06-18T03:52:47Z   <none>

local        local            v1.18.4+k3s1   k3s        2020-04-24T01:17:51Z   <none>

c-br8mf      staging-tenant   v1.18.4+k3s1   k3s        2020-06-19T18:43:23Z   <none>

Defaulting container name to rancher.

Use 'kubectl describe pod/rancher-bf4888754-4fkzq -n cattle-system' to see all of the containers in this pod.

Defaulting container name to rancher.

Use 'kubectl describe pod/rancher-bf4888754-4fkzq -n cattle-system' to see all of the containers in this pod.

--------------------------------------------------------------------------------

Cluster: d-testing (c-g8l7q)

Defaulting container name to rancher.

Use 'kubectl describe pod/rancher-bf4888754-4fkzq -n cattle-system' to see all of the containers in this pod.

Node Id   Address   Role   CPU   RAM   OS    Docker Version   Created

Node count: 0

Defaulting container name to rancher.

Use 'kubectl describe pod/rancher-bf4888754-4fkzq -n cattle-system' to see all of the containers in this pod.

--------------------------------------------------------------------------------

Cluster: local (local)

Defaulting container name to rancher.

Use 'kubectl describe pod/rancher-bf4888754-4fkzq -n cattle-system' to see all of the containers in this pod.

Node Id         Address                            Role     CPU   RAM         OS       Docker Version   Created

machine-mf2sg   10.100.101.26,ip-10-100-101-26     <none>   2     7850704Ki   <none>   <none>           2020-06-11T01:21:38Z

machine-gwjpb   10.100.100.123,ip-10-100-100-123   <none>   2     7936716Ki   <none>   <none>           2020-06-13T01:59:17Z

Node count: 2

Defaulting container name to rancher.

Use 'kubectl describe pod/rancher-bf4888754-4fkzq -n cattle-system' to see all of the containers in this pod.

--------------------------------------------------------------------------------

Cluster: staging-tenant (c-br8mf)

Defaulting container name to rancher.

Use 'kubectl describe pod/rancher-bf4888754-4fkzq -n cattle-system' to see all of the containers in this pod.

Node Id         Address                        Role     CPU   RAM         OS       Docker Version   Created

machine-zqzbb   10.10.100.4,ip-10-10-100-4     <none>   2     7850700Ki   <none>   <none>           2020-06-19T18:52:31Z

machine-znrrm   10.10.101.33,ip-10-10-101-33   <none>   2     7850708Ki   <none>   <none>           2020-06-19T18:52:31Z

Node count: 2

--------------------------------------------------------------------------------

Total node count: 4

msmtp: /etc/msmtprc: line 2: command host needs an argument
```
May because of typo in answers:
image

File: charts/systems-information/questions.yml [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+  required: true
+  group: "General Settings"
+- variable: send_to_support
+  default: "true"
+  description: "Would you like this report sent to Rancher Support?"
+  type: string
+  label: Send report to Rancher Support
+  required: false
+  group: "General Settings"
+- variable: smpt_host
```
Comment (dnoland1) [inline charts/systems-information/questions.yml:52]: Typo, I suspect host doesn't get passed properly to container due to this.

Comment (mattmattox) [reply charts/systems-information/questions.yml]: Fixed in systems-information-0.0.1-rc2.tgz

**Comment by mattmattox** [comment]
Fixed the "wait helm template" issue
@dnoland1 ready for review

**Pull request merged**